src/ImageBuilder/help/Stop-AzImageBuilderTemplate.md

```diff
@@ -31,29 +31,27 @@ Cancel the long running image build based on the image template
 
 ### Example 1: Stop image template creation
 ```powershell
-PS C:\> Start-AzImageBuilderTemplate -ImageTemplateName template-name-sn78hg -ResourceGroupName wyunchi-imagebuilder -AsJob 
+Start-AzImageBuilderTemplate -ImageTemplateName template-name-sn78hg -ResourceGroupName wyunchi-imagebuilder -AsJob 
 
 Id     Name            PSJobTypeName   State         HasMoreData     Location             Command
```

Comment:
move this output to Start-AzImageBuilderTemplate.md Example 1, then delete this output